napalm_junos/junos.py

```diff
@@ -331,6 +331,8 @@ def get_environment(self):
         for sensor_object, object_data in environment.items():
             structured_object_data = {k: v for k, v in object_data}
 
+            structured_object_data['class'] = structured_object_data['class'] or current_class
```

Comment:
Yeah, now with tests we are getting a CI error complaining that attribute is nit defined.

https://travis-ci.org/napalm-automation/napalm-junos/jobs/257353943#L406